docs/claims.md

```diff
@@ -0,0 +1,194 @@
+---
+id: claims
+title: Polkadot Claims
+sidebar_label: Claims
+---
+
+If you made a DOT purchase in one of the pre-launch sales, then you will need to claim your DOT
+tokens. For those who made a pre-genesis claim by sending a claim transaction to the Claims Contract
+on Ethereum, you will still need to send a free _attest_ transaction that will agree to the terms
+and conditions of your allocation. If you did not make a pre-genesis claim, then you will claim and
+attest in a single transaction.
+
+This guide will walk you through the steps for either making a claim or attesting to the statement.
+
+If you are making a claim on Polkadot for the first time please read on below in the
+[making a claim](#making-a-claim) section. If you've already claimed during the pre-genesis claims
+please proceed to [attesting to a statement](#attesting-to-a-statement) section instead.
+
+## Making a Claim
+
+If you did not make a claim in the pre-genesis claims period, then you are able to claim your DOT
+tokens after genesis. There is no time limit for making your claim so feel free to do this whenever
+you are most comfortable.
+
+> Note: When you make a claim, you will also attest to the agreement that corresponds to your DOT
+> allocation. The two actions "claim" and "attest" are done in a single transaction, but for the
+> most part this is simplified from the perspective of the UI.
+
+### What You Will Need
+
+- The Ethereum account that holds the DOT indicator tokens
+- The MyCrypto wallet
+- A Polkadot account
+
+You should already have your Ethereum account that holds the DOT indicator tokens from Polkadot's
+prior sales. You will need to have access to this account in order to make a signature.
+
+[MyCrypto][] is a versatile wallet that supports a variety of storage methods for your Ethereum
+account. Go to their downloads page and ensure that you download the latest version for your
```

Comment:
Should we include a link to their downloads page here?  Or say that it was linked previously?

Reply:
It is linked in the MyCrypto right there. But could just move that to "downloads page" in the text.

Reply:
Yeah, it was linked in the previous sentence, but to me this just looks awkward saying "go to their downloads page" without a link or reference.  

My preference would be to move the link to here.